src/win/error.c

```diff
@@ -168,6 +168,15 @@ int uv_translate_sys_error(int sys_errno) {
     case ERROR_NOT_SAME_DEVICE:             return UV_EXDEV;
     case ERROR_INVALID_FUNCTION:            return UV_EISDIR;
     case ERROR_META_EXPANSION_TOO_LONG:     return UV_E2BIG;
+    case WSAEBADF:                          return UV_EINVAL;
+    case WSAEDESTADDRREQ:                   return UV_EDESTADDRREQ;
+    case WSAEPROTOTYPE:                     return UV_EPROTOTYPE;
+    case WSAENOPROTOOPT:                    return UV_ENOPROTOOPT;
+    case WSAESTALE:                         return UV_EBADF;
```

Comment:
(GH ate my comment...)

This is arguably better handled by introducing a new `UV_ESTALE` error code (see 43c74a6ee92a8e646201eef1fd056799cdccbfde for an example) because `ESTALE` is an error that can pop up on other platforms too.